curriculum/challenges/english/25-front-end-development/workshop-blog-page/67d1f348f2cc9eab9d0b4ff4.md

```diff
@@ -0,0 +1,245 @@
+---
+id: 67d1f348f2cc9eab9d0b4ff4
+title: Step 23
+challengeType: 0
+dashedName: step-23
+---
+
+# --description--
+
+In previous lessons, we explored the `mailto` concept which enables users to open their default email client and compose an email by clicking an email link.
```

Comment:
`we` is not to be used. Refer to the style guide

Reply:
https://contribute.freecodecamp.org/how-to-work-on-coding-challenges/#challenge-descriptionsinstructions